app/src/main/java/io/left/meshim/controllers/RightMeshController.java

```diff
@@ -108,8 +112,10 @@ public void sendTextMessage(User recipient, String message) {
         try {
             byte[] messagePayload = createMessagePayloadFromMessage(messageObject);
             if (messagePayload != null) {
-                meshManager.sendDataReliable(recipient.getMeshId(), MESH_PORT, messagePayload);
-                dao.insertMessages(messageObject);
+                int deliveryDataID = meshManager.sendDataReliable(recipient.getMeshId(), MESH_PORT, messagePayload);
+               long insertedMessageInfo[] = dao.insertMessages(messageObject);
```

Comment:
Indentation is too far back one space here and in the line below.

Reply:
Ah! almost had a perfect PR haha